Intersect (Core)/Configuration/ClientConfiguration.cs

```diff
@@ -52,7 +52,20 @@ public static ClientConfiguration LoadAndSave(string filePath = null)
         public const long DEFAULT_MENU_BACKGROUND_FRAME_INTERVAL = 50;
 
         public const string DEFAULT_MENU_MUSIC = "RPG-Theme_v001_Looping.ogg";
+        
+        public const bool DEFAULT_TYPEWRITER_ENABLED = true;
 
+        public const long DEFAULT_TYPEWRITER_FULL_STOP_SPEED = 400;
+
+        public const long DEFAULT_TYPEWRITER_PART_DELAY = 6;
+
+        public const long DEFAULT_TYPEWRITER_PARTIAL_STOP_SPEED = 80;
```

Comment:
Still change PARTIAL_STOP/PartialStop to PAUSE/Pause

Reply:
Done